cmd/neofs-cli/modules/object/put.go

```diff
@@ -219,11 +223,25 @@ func parseObjectAttrs(cmd *cobra.Command) ([]object.Attribute, error) {
 		if len(kv) != 2 {
 			return nil, fmt.Errorf("invalid attribute format: %s", rawAttrs[i])
 		}
+
+		if kv[0] == object.AttributeTimestamp && !disableTime {
+			return nil, errors.New("can't override default timestamp attribute, use '--disable-timestamp' flag")
+		}
+
+		if kv[0] == object.AttributeFileName && !disableFilename {
+			return nil, errors.New("can't override default filename attribute, use '--disable-filename' flag")
+		}
+
```

Comment:
there are also lifetime flags `-e` or `-l` setting `__NEOFS__EXPIRATION_EPOCH` which can be `kv[0]`. They also must not cause duplication or conflict